components/apimgt/org.wso2.carbon.apimgt.impl/src/test/java/org/wso2/carbon/apimgt/impl/APIConsumerImplTest.java

```diff
@@ -1041,7 +1041,7 @@ public void testRequestApprovalForApplicationRegistration() throws APIManagement
         try {
             apiConsumer
                     .requestApprovalForApplicationRegistration("1", "app1", "access", "identity.com/auth", null, "3600",
-                            "api_view", "2", null, "default", null);
+                            "api_view", "2", null, "default", null, false);
```

Comment:
Do we need these changes as there's an overloaded method

Reply:
there's no need but orginally thought of deprecating the original method as it doesn't give much value to the interface.